README.md

```diff
@@ -59,100 +59,152 @@ build out any helper methods if needed.
-  - Names must be of type `str`
-  - Names must be between 1 and 15 characters, inclusive
-  - if you are using exceptions, uncomment lines 26-27 and 34-38 in
-    `customer_test.py`.
+- 
+  ```python
+  def __init__(self, name)
+  ```
+  - Customer should be initialized with a name 
+- 
+  ```python
+  @property
+  def name(self)
+  ```
+    - Returns the customer's name, as a string
+- 
+  ```python
+  @name.setter
+  def name(self, name)
+  ```
+    - Names must be of type `str`
+    - Names must be at least 1 character and at most 15 characters long
     - `raise Exception` if setter fails
+      
 
 #### Coffee
 
-- `Coffee __init__(self, name)`
+- 
+  ```python
+  def __init__(self, name)
+  ```
   - Coffees should be initialized with a name, as a string
-- `Coffee property name`
-  - Returns the coffee's name
-  - Should not be able to change after the coffee is created
-  - _hint: `hasattr()`_
-  - if you are using exceptions, uncomment lines 24-25 in `coffee_test.py`.
+- 
+  ```python
+  @property
+  def name(self)
+  ```
+    - Returns the coffee's name
+- 
+  ```python
+  @name.setter
+  def name(self, name)
+  ```
+    - Should not be able to change after the coffee is created
+      - _hint: `hasattr()`_
     - `raise Exception` if setter fails
 
 #### Order
 
-- `Order __init__(self, customer, coffee, price)`
+- 
+    ```python
+    def __init__(self, customer, coffee, price)
+    ```
   - Orders should be initialized with a customer, coffee, and a price (a number)
-- `Order property price`
-  - Returns the price for a coffee
-  - Price must be a number between 1 and 10, inclusive
+- 
+  ```python
+  @property
+  def price(self)
+  ```
+    - Returns the price for an order
+- 
+  ```python
+  @price.setter
+  def price(self, price)
+  ```
+    - Price must be at least 1 and no greater than 10
+    - `raise Exception` if setter fails
+- 
+  ```python
+  @property
+  def customer(self)
+  ```
+    - Returns the customer object for that order
+- 
+  ```python
+  @customer.setter
+  def customer(self, customer)
+  ```
+    - The argument `customer` must be of type `Customer`
+    - `raise Exception` if setter fails
+- 
+  ```python
+  @property
+  def coffee(self)
+  ```
+    - Returns the coffee object for that order
+- 
+  ```python
+  @coffee.setter
+  def coffee(self, coffee)
+  ```
+    - The argument `coffee` must be of type `Coffee` 
     - `raise Exception` if setter fails
 
-### Object Relationship Methods and Properties
-
-#### Order
+### Object Relationship Methods
 
-- `Order property customer`
-  - Returns the customer object for that order
-  - Must be of type `Customer`
-  - `raise Exception` if setter fails
-- `Order property coffee`
-  - Returns the coffee object for that order
-  - Must be of type `Coffee`
-  - `raise Exception` if setter fails
 
 #### Coffee
 
-- `Coffee orders(new_order=None)`
-  - Adds new orders to coffee
+- 
+  ```python
+  def orders(new_order=None)
+  ```
+  - Adds `new_order` to `Coffee`'s 
```

Comment:
This part trips me up a bit. It seems like it's saying a new order is being added to belong to the Coffee class itself. I don't like how it was before either - but maybe something like - Adds `new_order` to the coffee's orders

The greater problem here is we should be changing this so the coffees don't directly store a list of their orders anyway, to maintain a single source of truth, but that's another matter we'll need to fix in a different pr probably